udata_tabular_preview/preview.py

```diff
@@ -12,8 +11,8 @@ def preview_base_url(self):
 
     def can_preview(self, resource):
```

Comment:
Shouldn't we base `can_preview` on `analysis:parsing:finished_at` extras now following https://github.com/etalab/udata-hydra/pull/91?

Reply:
ok